.github/workflows/dodeploy.yml

```diff
@@ -0,0 +1,23 @@
+name: Sync to DigitalOcean Spaces
+
+on: 
+  push:
+    branches: 
+      - master
```

Comment:
@Miniland1333 Can we just change this to `main`? We don't have a master branch. Or we can actually keep both?

Reply:
I'm not sure at this time which one(s) will work since Github indicates that they are still in the middle of the transition from master to main. I believe what you are proposing will work but we will have to test to confirm.

Reply:
I'm of the opinion to leave everything as master until Github actually forces us to change. Having decades and hundreds of repos that have to be modified is a pain in the butt. Hopefully they will release some kind of tools or something that makes these transition a one line command or something.

Reply:
I believe moving branch names is only a couple of commands, although I understand there are other problems (like workflows or other services hardcoded to use a master branch). My opinion is there isn't a problem for new repos to start using a `main` branch instead of a `master` branch, so ckeditor-query-plugin can probably keep the main branch (one less repo to worry about for the future).

Reply:
In the mean time, the `main` branch is our default branch. We have no `master` branch at the moment. So we need `main` in the yaml file